src/commands.rs

```diff
@@ -96,9 +96,9 @@ impl Commands {
                         state = self.add_code_segment_multi_line(state, segment);
                     } else if segment.starts_with("```") && segment.ends_with("```") {
                         state = self.add_code_segment_single_line(state, 3, segment);
-                    } else if segment.starts_with("`") && segment.ends_with("`") {
+                    } else if segment.starts_with('`') && segment.ends_with('`') {
```

Comment:
Why?

Reply:
its a clippy lint against single character strings in string matching